src/XMLimport.cpp

```diff
@@ -149,7 +155,39 @@ bool XMLimport::importPackage(QIODevice* device, QString packName, int moduleFla
         readNext();
 
         if (isStartElement()) {
-            if (name() == "MudletPackage") {
+            if (name() == QStringLiteral("MudletPackage")) {
+                QString versionString;
+                if (attributes().hasAttribute(QStringLiteral("version"))) {
+                    versionString = attributes().value(QStringLiteral("version")).toString();
+                    if (!versionString.isEmpty()) {
+                        bool isOk = false;
+                        float versionNumber = versionString.toFloat(&isOk);
+                        if (isOk) {
+                            mVersionMajor = qFloor(versionNumber);
+                            mVersionMinor = qRound(1000.0 * versionNumber) - (1000 * mVersionMajor);
+                        }
+                        if (pVersionString) {
+                            *pVersionString = versionString;
+                        }
+                    }
+                }
+
+                if (mVersionMajor > 1
+                    /*||(mVersionMajor==1&&mVersionMinor)*/) {
+                    // Minor check is not currently relevant, just abort on 2.000f or more
+
+                    QString moanMsg = tr("[ ALERT ] - Sorry, the file being read:\n"
+                                         "\"%1\"\n"
+                                         "reports it has a version (%2) it must have come from a newer Mudlet version,\n"
+                                         "which this one cannot read it, you need to update! The creators of Mudlet\n"
+                                         "suggest that you seek help and an update on-line using the links in the panel\n"
```

Comment:
My thoughts were that if they are getting this message it means that there has been a significant change - because I would hope that we would have included migration steps in Mudlet versions with minor number increments and if the user was a keen/frequent user they would already have adopted a later version - therefore someone getting this is not likely to be as familiar with the product and advising them of how/where to go to fix the issue is more friendly than just saying "Sorry Dude, you shouldn't be doing that with this!"

OTOH perhaps I am over-thinking this, so: "delete that last sentence" + previous sentence replace "to update" with "a newer Mudlet!" yes / no?